languages/rust/oso/src/host/to_polar.rs

```diff
@@ -172,3 +172,19 @@ impl<C: ToPolarIter> ToPolarIter for Option<C> {
         )
     }
 }
+
+pub struct PolarValues<I, Iter>
+where
+    I: ToPolarIter + 'static,
```

Comment:
Fun stuff. Should we call `ToPolarIter` something else?

Could revisit this separately.  If we never use it for more than one result, it seems like a trait along the lines of `to_polar_result() -> oso::Result<Option<Value>>` would be all we need?

Reply:
It is used for more than one thing now with the iterator stuff. I changed the names to hopefully be better.